### PR TITLE
Revised README for CloudWatch sample

### PR DESCRIPTION
**Pull request by davidbass13**

### CODE REVIEW

File: samples/cloudwatch/README.md [hunk 1]
```diff
@@ -1,73 +1,88 @@
...
+## CloudWatch Agent
+
+AWS uses the CloudWatch agent to collect metrics. There are two types of CloudWatch agents that you might want to use in your environment:
+
+- The CloudWatch agent with Prometheus monitoring scrapes metrics from Prometheus sources, such as SAS Viya. The metrics can then be converted and mapped for display in CloudWatch. Because SAS Viya monitoring uses Prometheus, this sample explains how to deploy and configure only the CloudWatch agent for use with Prometheus. 
+
+- The default CloudWatch agent collects metrics from AWS nodes. Although these metrics are not specific to SAS Viya, you might want to use both agents so you can obtain a complete view of your environment's performance. For information about using the default agent, see the 
+[CloudWatch documentation](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/Install-CloudWatch-Agent.html). If you do use the default agent, you must add tolerations to the
+agent's daemonset definition so that the agent can run on nodes that are tainted to support SAS Viya workload node placement. See [CloudWatch Agent Daemonset and Workload Node Placement](#wnp_tolerations).
```
Comment (gsmith-sas) [inline samples/cloudwatch/README.md:16]: Should we make it clear that this is pointer to elsewhere in the same document? I wasn't sure where the link was going to take and assumed it was a whole separate document.  Perhaps: _See the [blah blah] section below for more information._

File: samples/cloudwatch/README.md [hunk 2]
```diff
@@ -1,73 +1,88 @@
...
-covered extensively in the online
-[CloudWatch documentation](https://aws.amazon.com/cloudwatch/getting-started/).
+## CloudWatch Agent
+
+AWS uses the CloudWatch agent to collect metrics. There are two types of CloudWatch agents that you might want to use in your environment:
+
+- The CloudWatch agent with Prometheus monitoring scrapes metrics from Prometheus sources, such as SAS Viya. The metrics can then be converted and mapped for display in CloudWatch. Because SAS Viya monitoring uses Prometheus, this sample explains how to deploy and configure only the CloudWatch agent for use with Prometheus. 
+
+- The default CloudWatch agent collects metrics from AWS nodes. Although these metrics are not specific to SAS Viya, you might want to use both agents so you can obtain a complete view of your environment's performance. For information about using the default agent, see the 
+[CloudWatch documentation](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/Install-CloudWatch-Agent.html). If you do use the default agent, you must add tolerations to the
```
Comment (gsmith-sas) [inline samples/cloudwatch/README.md:15]: Is "must" correct here?  Is WNP optional or required on AWS?

**Review by gsmith-sas — COMMENTED** [review]
Looks good.  I wasn't sure if WNP is required on AWS or not; will leave to Bryan to clarify.

File: samples/cloudwatch/README.md (hunk 2 above)
Comment (BryanEllington) [reply samples/cloudwatch/README.md:15]: WNP is definitely enabled by default by IAC. I think it is required on some level - at least for CAS - so this step will be necessary regardless of whether it's enabled for stateful, stateless, etc.

File: samples/cloudwatch/README.md (hunk 1 above)
Comment (BryanEllington) [reply samples/cloudwatch/README.md:16]: It's normal to link to sections of a document. Not sure what the issue is here. Click link, get info - regardless of whether it's on the same page or not.

File: samples/cloudwatch/README.md [hunk 3]
```diff
@@ -1,73 +1,88 @@
...
-# Scraped metrics will become performance log events
 kubectl apply -f sas-prom-config.yaml
-# Map performance log events to CloudWatch metrics
+```
+#### Map Performance Log Events to CloudWatch Metrics
+
+Run these commands to map the performance log events that were created from Prometheus metrics to CloudWatch metrics and restart the agent in order to apply the configuration changes:
+
+```bash
 kubectl apply -f sas-prom-cwagentconfig.yaml
```
Comment (BryanEllington) [inline samples/cloudwatch/README.md:86]: Remove this line to eliminate the merge conflict. We don't need to apply the yaml as is. The next line applies the same yaml, but 'fixed' with the correct replacement values.

File: samples/cloudwatch/README.md [hunk 4]
```diff
@@ -76,34 +91,29 @@ cat sas-prom-cwagentconfig.yaml \
...
-* Search for or find the `go_memstates_alloc_bytes` metric
-* Right-click and choose `Search for this only`
-* Select the checkbox of one or more services (rows) to add them
-to the graph at the top
+1. From the AWS web console, navigate to CloudWatch.
+2. In the navigation bar on the left, choose `Metrics`.
+3. Select the `ContainerInsights/Prometheus` namespace.
+4. Select `ClusterName, job, namespace, pod, sas_service_base`.
+5. Locate the `go_memstates_alloc_bytes` metric.
+6. Right-click on the metric and select `Search for this only`.
```
Comment (BryanEllington) [inline samples/cloudwatch/README.md:107]: It's a left-click on the down-arrow next to the metric name.

File: samples/cloudwatch/README.md [hunk 5]
```diff
@@ -76,34 +91,29 @@ cat sas-prom-cwagentconfig.yaml \
...
-* Choose `ClusterName, job, namespace, pod, sas_service_base`
-* Search for or find the `go_memstates_alloc_bytes` metric
-* Right-click and choose `Search for this only`
-* Select the checkbox of one or more services (rows) to add them
-to the graph at the top
+1. From the AWS web console, navigate to CloudWatch.
+2. In the navigation bar on the left, choose `Metrics`.
+3. Select the `ContainerInsights/Prometheus` namespace.
+4. Select `ClusterName, job, namespace, pod, sas_service_base`.
+5. Locate the `go_memstates_alloc_bytes` metric.
```
Comment (BryanEllington) [inline samples/cloudwatch/README.md:106]: The correct metric name is go_memstats_alloc_bytes

File: samples/cloudwatch/README.md (hunk 1 above)
Comment (davidbass13) [reply samples/cloudwatch/README.md:16]: I agree with Bryan - I don't think a notice is needed that the link goes to the same document.

File: samples/cloudwatch/README.md (hunk 3 above)
Comment (davidbass13) [reply samples/cloudwatch/README.md:86]: Line removed

File: samples/cloudwatch/README.md (hunk 4 above)
Comment (davidbass13) [reply samples/cloudwatch/README.md:107]: Text changed

File: samples/cloudwatch/README.md (hunk 5 above)
Comment (davidbass13) [reply samples/cloudwatch/README.md:106]: Changed

**Review by BryanEllington — APPROVED** [review]
Looks good

**Pull request merged**